scripts/data/prompt.txt

```diff
@@ -22,13 +22,14 @@ COMMANDS:
 15. Get Improved Code: "improve_code", args: "suggestions": "<list_of_suggestions>", "code": "<full_code_string>"
 16. Write Tests: "write_tests", args: "code": "<full_code_string>", "focus": "<list_of_focus_areas>"
 17. Execute Python File: "execute_python_file", args: "file": "<file>"
-18. Task Complete (Shutdown): "task_complete", args: "reason": "<reason>"
+18. Scientific Research: "arxiv_search", args: "input": "<search>"
+19. Task Complete (Shutdown): "task_complete", args: "reason": "<reason>"
 
 RESOURCES:
 
 1. Internet access for searches and information gathering.
 2. Long Term memory management.
-3. GPT-3.5 powered Agents for delegation of simple tasks.
+3. GPT-4 powered Agents for delegation of simple tasks.
```

Comment:
Can you explain this change?

Reply:
Low consumption reasons. This `agi` burns the wallet if gpt-4 is used even for simple tasks. No?